### PR TITLE
New package: python-pygame-1.9.1

### PR DESCRIPTION
**Pull request by archaeme**
Also added smpeg to common/shlibs to satisfy dependencies.

### CODE REVIEW

File: srcpkgs/python-pygame/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+wrksrc="pygame-${version}release"
+build_style=python-module
+short_desc="Collection of python modules for writing games"
+maintainer="Archaeme <normandy@firemail.cc>"
+license="LGPL-2.1"
+homepage="http://www.pygame.org/"
+distfiles="http://www.pygame.org/ftp/pygame-${version}release.tar.gz"
+checksum="a26095472ae4be9631e0d5bfb9a52ac57a3a091e45757913128e4a473807d433"
+
+depends="python sdl_mixer sdl_image sdl_ttf smpeg"
```
Comment (Vaelatern) [inline srcpkgs/python-pygame/template]: This line is not necessary

**Comment by Vaelatern** [comment]
Please rebase off of master and force push to start a Travis test.

**Comment by archaeme** [comment]
Ok, made some changes to get the build to pass and to make it work on ARM.

File: srcpkgs/python-pygame/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+revision=1
+wrksrc="pygame-${version}release"
+build_style=python-module
+short_desc="Collection of python modules for writing games"
+maintainer="Archaeme <normandy@firemail.cc>"
+license="LGPL-2.1"
+homepage="http://www.pygame.org/"
+distfiles="http://www.pygame.org/ftp/pygame-${version}release.tar.gz"
+checksum="a26095472ae4be9631e0d5bfb9a52ac57a3a091e45757913128e4a473807d433"
+hostmakedepends="python-devel SDL_mixer-devel SDL_image-devel SDL_ttf-devel smpeg-devel v4l-utils-devel"
```
Comment (Duncaen) [inline srcpkgs/python-pygame/template]: Those are all `makedepends`, only `python` would be `hostmakedepends`.

Comment (archaeme) [inline srcpkgs/python-pygame/template]: Gonna add `smpeg-devel` to the `hostmakedepends`, otherwise ARM compilation fails.

**Pull request closed without merging**